app/secrets.py

```diff
@@ -20,8 +24,8 @@ def validate_required_secrets(secrets, additional_required_secrets=None):
 
 
 class SecretStore:
-    def __init__(self, secrets):
+    def __init__(self, secrets: dict) -> None:
         self.secrets = secrets.get("secrets", {})
 
-    def get_secret_by_name(self, secret_name):
+    def get_secret_by_name(self, secret_name: str) -> Any:
```

Comment:
Isn't this returning a string?

Reply:
Same as above I did try this but it errored saying that it is expecting to return an any so I set it to any

Reply:
Would be fixed once # is addressed.